lib/utils.pm

```diff
@@ -1218,7 +1218,7 @@ sub ensure_serialdev_permissions {
     # reboot an alternative https://superuser.com/a/609141/327890 would need
     # handling of optional sudo password prompt within the exec
     # Need backwards support for SLES11-SP4 here, the command "gpasswd" and "stat" are only available with SLES-12 at least.
-    if (is_sle && check_var('VERSION', '11-SP4')) {
+    if (is_sle && (check_var('VERSION', '11-SP4') || check_var('VERSION', '12-SP5'))) {
```

Comment:
It is not clear, why we need this change. Seems like an old code.

Reply:
invalid comment. as the PR is still draft is keep changing. feel free to reopen it when you find it at the latest commit.